### PR TITLE
You can rekey screens (inbox and convo list)

### PR DESCRIPTION
**Pull request by chrisnojima**
This does the 'you can rekey' screens for chat. also fixed the centering thing with the participants rekey list

@keybase/react-hackers

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
(note to send i had a master -> branch -> branch -> branch and merged down the middle two here: https://github.com/keybase/client/pull/5597) so likely i should wait till https://github.com/keybase/client/pull/5592 is in, then change this to be based off master)

File: shared/chat/conversation/index.desktop.js [hunk 1]
```diff
@@ -175,7 +176,11 @@ export default branch(
   renderComponent(NoConversation),
   branch(
     (props: Props) => !!props.rekeyInfo,
-    renderComponent(ParticipantRekey),
+    branch(
```
Comment (chromakode) [inline shared/chat/conversation/index.desktop.js]: For general discussion: I think this is getting really confusing with the nesting. Took me way too long to figure out what this branching was doing and what  `withFocusHandlers` applied to. What is the advantage of expressing the logic in this way vs. putting it in the caller?

Comment (chrisnojima) [inline shared/chat/conversation/index.desktop.js]: Yeah this thing is creaky. Likely better to make a new component in this file to express this. I don't think the connected component should know about these rendering details though. Luckily this is the last of these branching paths so

**Pull request merged**